stories/card/card.stories.js

```diff
@@ -0,0 +1,719 @@
+
+**Content:**
+
+- reserved for application content.
+`
+    }
+};
+
+export const analyticalCard = () => `
+<div style="display:flex; justify-content:space-around; flex-wrap: wrap">
+    <div style="width: 300px; height: 400px; margin: 1rem;">
+        <div class="fd-card fd-card--analytical" role="region" aria-label="analytical-card-1">
+            <a class="fd-card__header" tabindex="0" role="heading" aria-level="3" aria-roledescription="Card Heading" aaria-label="card-header">
+                <div class="fd-card__header-text">
+                    <div class="fd-card__title-area">
+                        <div class="fd-card__title">Card Title</div>
+                        <span class="fd-object-status fd-card__counter">Counter</span>
+                    </div>
+                    <div class="fd-card__subtitle-area">
+                        <div class="fd-card__subtitle">Card Subtitle</div>
+                        <div class="fd-card__currency">Currency</div>
+                    </div>
+                    <div class="fd-card__analytics-area">
+                        <div class="fd-numeric-content">
+                            <div class="fd-numeric-content__kpi-container">
+                                <div class="fd-numeric-content__kpi">1Ñç</div>
+                            </div>
+                            <div class="fd-numeric-content__scale-container">
+                                <div class="fd-numeric-content__scale">
+                                    <span class="fd-numeric-content__scale-arrow sap-icon--down"></span>
+                                    <span class="fd-numeric-content__scale-text">M</span>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="fd-card__analytics-container">
+                            <div class="fd-card__analytics">
+                                <div class="fd-card__analytics-text">Target</div>
+                                <div class="fd-card__analytics-content">110K</div>
+                            </div>
+                            <div class="fd-card__analytics">
+                                <div class="fd-card__analytics-text">Deviation</div>
+                                <div class="fd-card__analytics-content">-35.7%</div>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="fd-card__second-subtitle">Second Subtitle</div>
+                </div>
+            </a>
+            <div class="fd-card__content" role="group" aria-label="Card Content"></div>
+        </div>
+    </div>
+
+    <div style="width: 700px; height: 400px; margin: 1rem;">
+        <div class="fd-card fd-card--analytical"  role="region" aria-label="analytical-card-2">
+            <a class="fd-card__header" tabindex="0" role="heading" aria-level="3" aria-roledescription="Card Heading" aaria-label="card-header">
```

Comment:
Typo: `aaria-label`

There are a few a11y issues with the link. First an aria-label should be more descriptive than something like 'card-header' but also, an aria-label on a link like this will override the entire rest of the content once tabbed to. So a screenreader user won't get any of the content laid out below. Would it be better to enclose just the title as a link? Then the rest of the content could be read as normal.

An `a` does not need an `aria-roledescription` as it already has a predefined role of `link` that we shouldn't be trying to override as well.

Reply:
Hi @meganaconley , the entire header should be clickable as specified in the specs and the UX guidelines. I see UI5 used a normal `div` not an `a`. Would it be better to use a `div`?

<img width="437" alt="Screen Shot 2020-08-13 at 9 24 04 AM" src="https://user-images.githubusercontent.com/39598672/90139799-d50bf000-dd46-11ea-82f3-61f91cfcfcd6.png">